src/Components/login/Register.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState, ReactElement, SyntheticEvent } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  dispatchToken,
+  dispatchUserObject,
```

Comment:
Rename folder structure so the Register component isn't inside a folder called login. We need to maintain clear folder and file paths.